.github/actions/install-risc0/action.yml

```diff
@@ -20,7 +20,7 @@ runs:
       # visit https://github.com/risc0/risc0 and https://github.com/risc0/rust
       # release pages.
       run: |
-        cargo binstall -y rzup \
+        cargo install rzup --version 0.4.1 --locked \
```

Comment:
Can you also change the step name? It still refers to `cargo-binstall`.

Reply:
Also, we can probably remove the cargo-binstall installation step, right?

Reply:
Will update the name, but rzup uses `cargo-binstall`, we still need `binstall`.